src/pdsfile/holdings_maintenance/_archives_common.py

```
@@ -0,0 +1,241 @@
+            logger.ds_store('.DS_Store file skipped', member.name)
+            return None
+
+        if basename.startswith('._') or '/._' in member.name:
+            logger.dot_underscore('._* file skipped', member.name)
+            return None
+
+        if basename.startswith('.') or '/.' in member.name:
+            if archive_invisibles:
+                logger.invisible('Invisible file archived', member.name)
+                return member
+            else:
```

Ruff (0.16.1)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)